OracleFMWInfrastructure/dockerfiles/12.2.1.3/Dockerfile

```diff
@@ -1,6 +1,6 @@
-#Copyright (c) 2014-2018 Oracle and/or its affiliates. All rights reserved.
+#Copyright (c) 2014, 2020, Oracle and/or its affiliates.
 #
```

Comment:
Why the empty line here? There are a few other instances like this. Later in the review, new files don't have an empty line here.

Reply:
These are not new files but updated but I can remove the empty line np.